docs/markdown/glossary.md

```diff
@@ -0,0 +1,256 @@
+# Glossary
+
+This glossary provides definitions for key terms in the Interledger ecosystem.
+
+## Account
+   
+An accounting relationship between two Interledger participants. In Interledger, two participants establish an account with one another to track the current obligations they hold with one another. An Interledger account has a [balance](#balance) denominated in a mutually agreed upon asset (e.g. USD) at an agreed upon scale (e.g. 2).                                                                        
```

Comment:
Since "scale" is not defined here (and AFAIK is a concept we invented), maybe replace "scale" with "denomination"?